docs/charts/index.yaml

```diff
@@ -1,53 +1,53 @@
 apiVersion: v1
 entries:
   weblogic-operator:
-  - created: "2019-11-15T20:34:35.377589-05:00"
+  - created: "2020-01-15T20:11:19.500388682Z"
     description: Helm chart for configuring the WebLogic operator.
-    digest: b36bd32083f67453a62d089a2c09ce38e6655d88ac8a7b38691230c55c40e672
+    digest: d358882813647586f6137061de441c4b89c775dcda6653e8cc6e6cf2222cb48b
     name: weblogic-operator
     urls:
     - https://oracle.github.io/weblogic-kubernetes-operator/charts/weblogic-operator-2.4.0.tgz
     version: 2.4.0
-  - created: "2019-11-15T20:34:35.37493-05:00"
+  - created: "2020-01-15T20:11:19.499534741Z"
```

Comment:
why are you changing the old ones?

Reply:
I believe this is auto-generated.

Reply:
yes, this file gets updated automatically.